packaging/docker/pom.xml

```diff
@@ -13,7 +13,7 @@
     <properties>
         <dependency.alfresco-mmt.version>6.0</dependency.alfresco-mmt.version>
         <!-- Alfresco GoogleDocs integration version -->
-        <alfresco.googledocs.version>3.1.0</alfresco.googledocs.version>
+        <alfresco.googledocs.version>3.2.0-A4</alfresco.googledocs.version>
```

Comment:
In 3.2 the artifact names have been changed. The new artifact is called `alfresco-googledrive-share`. See line 37

Reply:
Yes, as per https://github.com/Alfresco/googledrive/blame/master/_ci/prepare_staging_deploy.sh#L28-L29(GOOGLEDOCS-464).